Translate Property Editor Values in a multilingual site

eshanrnh opened the pull request:
## Description

Some property editors can be set up with multiple values, this section documents how to translate these values using Dictionary Items in a multilingual setup.

I added this section in the Creating a Multilingual Site Tutorial since the [Dictionary Items section](https://docs.umbraco.com/umbraco-cms/fundamentals/data/dictionary-items#using-dictionary-item-in-a-multilingual-website) links back to the tutorial.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

CMS v13 and v14

## Deadline (if relevant)

Anytime

sofietoft commented on 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/radiobutton-list.md:
Could it make sense to add the info about how to translate the prevalues close to the "Data Type Definition example"? And then perhaps in a note instead?

sofietoft commented on 13/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/checkbox-list.md:
```diff
@@ -102,3 +102,7 @@ If Modelsbuilder is enabled you can get the alias of the desired property withou
+
+## Translating Checkbox List Values Using Dictionary Items
+
+To translate the values of a Checkbox List property editor in a multi-lingual setup, you can use Dictionary items. For more details, see the [Translating Multi-Value Property Editors](../../../../tutorials/multilanguage-setup.md#translating-multi-value-property-editors) section in the [Creating a Multilingual Site](../../../../tutorials/multilanguage-setup.md) article.
```
Instead of two links to the same article, I think it's enough using the anchor link with the title of the article 💪 

```suggestion
You can use Dictionary items to translate the values of a Checkbox List property editor in a multilingual setup. For more details, see the [Translating Multi-Value Property Editors](../../../../tutorials/multilanguage-setup.md#translating-multi-value-property-editors) section in the [Creating a Multilingual Site](../../../../tutorials/multilanguage-setup.md) article.
```

sofietoft commented on 13/umbraco-cms/tutorials/multilanguage-setup.md:
```diff
@@ -198,6 +198,38 @@ To render dictionary items in the template, replace the text with the following
 
+### Translating Multi-Value Property Editors
+
+When building a multilingual website in Umbraco, you can use property editors like Dropdown, Checkbox List, and Radiobutton List to offer content in multiple languages. By leveraging dictionary items, you can ensure that these property editors display the appropriate translated values based on the site's current language.
```
I think this first sentence makes it sound like ONLY these three property editors can offer content in multiple languages 🙈 

Perhaps something like this instead:

```suggestion
Property editors like Dropdown, Checkbox List, and Radiobutton List let you define a set of prevalues to be chosen between in the Content section. By leveraging dictionary items, you can ensure that these prevalues are translated correctly based on the site's current language.
```

sofietoft reviewed (changes requested):
I've added a couple of suggestions here and there.
Also, some general comments:

- Be consistent with how you capitalize Property Editors / property editors - the same goes for Dictionary Item / Dictionary item / dictionary item, etc. (perhaps we should add these to the "Umbraco Terms" styleguide rule)
- Be consistent with how multilingual is written
- Are we calling the prevalues _prevalues_ or _options_? Step 1 uses prevalues and Step 2 uses options.

I hope it all makes sense 🙌

eshanrnh commented on 13/umbraco-cms/tutorials/multilanguage-setup.md:
````diff
@@ -198,6 +198,38 @@ To render dictionary items in the template, replace the text with the following
+    <span>@translatedValue</span>
+    ```
+
+    The code retrieves the selected value from the property editor and translates it using the corresponding Dictionary item.
````
Yes cz it's part of the code/step.

sofietoft replied:
Ah! Makes sense then 😄

eshanrnh commented:
Thanks for your review, @sofietoft 🙌 All the comments/suggestions make perfect sense 😀

- I've updated "property editors", "dictionary item", and "multilingual" to lowercase except for when they are used in the heading or at the beginning of a sentence.
- Good catch. It's _prevalues_ in v13 and _options_ in v14 🤷‍♀️. I've updated both articles to reflect the appropriate UI term. 

Let me know if I missed anything.

sofietoft reviewed (commented):
Ah! I wasn't aware of the name change from "prevalues" to "options". That's good to know 💪 I'll create a task to get this updated throughout the docs!

I realize now that I should have been more clear regarding the capitalization of the Umbraco Terms. Ideally they should be written with capital letters, so "Property Editor", "Dictionary Item" - they are names of terms, and that's also how they're written in the backoffice UI 💪

croban commented:
is there a possibility to translate prevalues with translation files?

sofietoft commented:
I believe you should use dictionary items to translate the prevalues/options: https://docs.umbraco.com/umbraco-cms/tutorials/multilanguage-setup#translating-multi-value-property-editors

sofietoft commented:
That is a good question 🤔 
I'm not sure to me honest.

You might be better off heading to the [Umbraco community forum](https://forum.umbraco.com/) - I'm sure somewhere there can help you out! 💪

croban commented:
@sofietoft, sorry but on community I find answers rarely 😔, but thanks anyway.
